sql/catalyst/src/main/java/org/apache/spark/sql/vectorized/ArrowColumnVector.java

```diff
@@ -38,6 +43,29 @@
   private final ArrowVectorAccessor accessor;
   private ArrowColumnVector[] childColumns;
 
+  public static ArrowColumnVector[] loadColumns(StructType schema,
+                                                ArrowRecordBatch recordBatch) {
```

Comment:
I'm not really sure this makes sense here, it seems like it's mainly to convert an `ArrowRecordBatch` to Spark `ColumnarBatch`. That is basically what's done in `ArrowConverters.fromBatchIterator`, maybe you could move that to it's own function there?

Reply:
@BryanCutler, yes, the codes is very similiar because that is how arrow designed to load ArrowRecordBatch to java List[Vector], do you mean I should put this loadColumns function to ArrowConverters, and named as something like <code>ArrowConverters.toColumnarBatchIterator : Iterator[ColumnarBatch]</code> ? I am OK to do that, I just felt it could fit in both scope, in <code>ArrowConverters</code> or in <code>ArrowColumnVector</code>

Reply:
I was more suggesting something like adding `ArrowConverters.toColumnarBatch(recordBatch: ArrowRecordBatch): ColumnarBatch`. That is already basically one internally in `ArrowConverters.fromBatchIterator`, so instead of duplicating that same conversion just move the existing it to a function if it fits your needs too.